schema.graphql

```diff
@@ -6,7 +6,7 @@ type FactoryEventsSummary {
   moduleTemplateCount: BigInt!
   moduleCloneCount: BigInt!
   contestBuiltCount: BigInt!
-  ContestCloneCount: BigInt!
```

Comment:
I'm not sure if this was intentional but you may not want it as it will change queries that use this